tests/cpp/multidevice.h

```diff
@@ -33,30 +34,37 @@ class MultiDeviceEnvironment : public testing::Environment {
     return do_barrier_at_test_;
   }
 
-  bool disableSkip() const {
-    return disable_skip_;
-  }
-
  private:
   std::unique_ptr<Communicator> communicator_ = nullptr;
   bool debug_print_ = false;
   bool do_barrier_at_test_ = false;
-  bool disable_skip_ = false;
 };
 
 class MultiDeviceTest : public NVFuserTest {
  public:
+  // Given an aten tensor, TensorView the tensor is bound to, and deviceId
+  // returns a shard of the tensor according the sharding annotation in tv
+  // for the deviceId. If tensor is not sharded returns the original tensor.
+  // TODO: If deviceId is not part of the mesh this should return an empty
+  // tensor currently, we don't support this, so for now it returns a slice.
```

Comment:
for what reason don't we support that?
also, nit: add a "." before currently maybe ?

Reply:
Global input tensor sizes are used for concretization. We don't have a way to specify global input size separate from the input tensor yet.

Reply:
Oh right! I see